src/main/resources/application.yaml

```diff
@@ -7,6 +7,9 @@ spring:
     jpa.properties.hibernate:
         dialect: org.hibernate.dialect.PostgreSQLDialect
         jdbc.lob.non_contextual_creation: true
+        jdbc.batch_size: ${BATCH_SIZE}
```

Comment:
Why would we want this configurable?

Reply:
While I was trying batch updates with different batch sizes, I noticed that the batch size needs to be picked very carefully.
As per my understanding, increasing the batch size wouldn't necessarily improve the performance and the same for decreasing the batch size. So, I thought the batch size cannot be constant because it really varies with the number of objects we have to update, and that is why I made it configurable.

But since the update center rarely adds or deletes a new plugin, I guess we can keep it constant if we like. What do you think?

Reply:
I'd prefer to have a fix value because here the application would fail to start without the variable.